py/tests/advanced/algorithms/advanced_connections_test.py

```diff
@@ -38,207 +38,6 @@ def _getPresynapticCells(self, connections, segment, threshold):
         return set([connections.presynapticCellForSynapse(synapse) for synapse in connections.synapsesForSegment(segment) 
                     if connections.permanenceForSynapse(synapse) >= threshold])
 
-    def testAdaptShouldNotRemoveSegments(self):
-        """
-        Test that connections are generated on predefined segments.
```

Comment:
FYI @fcr I'm removing the "duplicit" tests for `advanced_connections.py` for methods that are not changed by this Connection implementation, and are already tested in the "main" tests. As I made changes to the tests and would have to port them here.